app/containers/HomePage.jsx

```diff
@@ -6,16 +6,25 @@ import * as actions from '../redux/actions/index';
 import adb from '../utils/adb';
 import AppBar from 'material-ui/AppBar';
 import { push } from 'react-router-redux';
+import result from 'lodash/result';
 import HeaderIcons from '../components/Header/Buttons';
 
 class HomePage extends Component {
+  autoRefresh = () => {
+    setInterval(() => {
+      if (result(this.props.state, 'devices.devicesList', []).length === 0) {
```

Comment:
This should maybe run even if there is already a device. I mean we can store the array lenght in a localvariable and whenever that changes , we can run the listDevices. so that way we can extend this feature to detect multiple devices